corehq/apps/locations/fixtures.py

```diff
@@ -241,20 +240,28 @@ def _get_children(domain, root, expand_to_level):
     return children
 
 
-def _get_include_without_expanding_locations(domain, location_type):
+def _get_include_without_expanding_locations(domain, assigned_locations):
     """returns all locations set for inclusion along with their ancestors
     """
-    include_without_expanding = location_type.include_without_expanding
-    if include_without_expanding is not None:
-        forced_location_level = (SQLLocation.active_objects
-                                 .filter(domain__exact=domain, location_type=include_without_expanding)
-                                 .values_list('level', flat=True)
-                                 .first())
-        if forced_location_level is not None:
-            return (SQLLocation.active_objects
-                    .filter(domain__exact=domain,
-                            level__lte=forced_location_level))
-    return SQLLocation.objects.none()
+    # all loctypes to include, based on all assigned location types
+    location_types = (LocationType.objects
+                      .filter(sqllocation__in=assigned_locations)
+                      .exclude(include_without_expanding=None)
+                      .distinct()
+                      .values_list('include_without_expanding', flat=True))
+    # all levels to include, based on the above loctypes
+    forced_levels = (SQLLocation.active_objects
+                     .filter(domain__exact=domain,
+                             location_type__in=location_types)
+                     .values_list('level', flat=True)
+                     .order_by('level')
+                     .distinct('level'))
+    if forced_levels:
+        return (SQLLocation.active_objects
+                .filter(domain__exact=domain,
+                        level__lte=max(forced_levels)))
```

Comment:
this only needs to be called once, with the max level, not once for each forced level
```
locations_lte3 | locations_lte2 == locations_lte3
```